code/modules/food_and_drink/snacks.dm

```diff
@@ -412,6 +412,10 @@
 			if (ispug(M))
 				..()
 				boutput(M, "<span class='notice'>That tasted delicious!</span>")
+			if (iswerewolf(M))
+				..()
+				boutput(M, "<span class='notice'>That tasted delicious!</span>")
+
```

Comment:
Gross, duplicate code

Reply:
Still new to this sorry, realised how bad that reads